ansible_events_ui/db/migrations/versions/2022_08_16_2afab2d669fe_.py

```diff
@@ -0,0 +1,70 @@
+"""Adds user roles.
```

Comment:
Can you please regenerate it or fix naming after #47 is merged? #47 changes the naming schema for migrations.

Reply:
Done